kubetest2-gke/deployer/up.go

```diff
@@ -217,9 +219,7 @@ func (d *deployer) verifyUpFlags() error {
 	if len(d.projects) == 0 && d.boskosProjectsRequested <= 0 {
 		return fmt.Errorf("either --project or --projects-requested with a value larger than 0 must be set for GKE deployment")
 	}
-	if err := d.verifyNetworkFlags(); err != nil {
-		return err
-	}
+
 	if len(d.clusters) == 0 {
 		if len(d.projects) > 1 || d.boskosProjectsRequested > 1 {
 			return fmt.Errorf("explicit --cluster-name must be set for multi-project profile")
```

Comment:
Let's move these checks above as well.

Reply:
Hmm not sure if I got your point, could you elaborate?

Reply:
nvm, I think I saw the diff wrong. you **are** moving the length check above verifyNetworkFlags